tests/integration/transactions/test_check_cancel.py

```diff
@@ -28,3 +30,16 @@ def test_all_fields(self):
         # ID field such as CheckID, Channel, Unauthorize."
         self.assertEqual(response.result["engine_result"], "tecNO_ENTRY")
         WALLET.sequence += 1
+
+    def test_high_fee_unauthorized(self):
+        # GIVEN a new CheckCancel transaction
+        check_cancel = CheckCancel(
+            account=ACCOUNT,
+            sequence=WALLET.sequence,
+            check_id=CHECK_ID,
+            # WITH fee higher than 2 XRP
+            fee=FEE,
+        )
+        # We expect an XRPLException to be raised
+        with self.assertRaises(XRPLException):
+            submit_transaction(check_cancel, WALLET)
```

Comment:
There doesn't need to be a test for every single transaction type - one test should be enough

Reply:
I removed all of them except two: AccountDelete and AccountSet